bin/network.js

```diff
@@ -1,15 +1,13 @@
-import { dappStoreUiUrlPreview, dappStoreUiUrlPreprod, dappStoreUiUrlMainnet } from './config.js';
-
 export const availableNetwork = {
   preview: 'preview',
   preprod: 'preprod',
   mainnet: 'mainnet',
 };
 
 const mapOfUIUrls = {
-  [availableNetwork.preview]: dappStoreUiUrlPreview,
-  [availableNetwork.preprod]: dappStoreUiUrlPreprod,
-  [availableNetwork.mainnet]: dappStoreUiUrlMainnet,
+  [availableNetwork.preview]: 'https://live-preview.ui.dapp-store.lw.iog.io',
+  [availableNetwork.preprod]: 'https://live-preprod.ui.dapp-store.lw.iog.io',
+  [availableNetwork.mainnet]: '',
```

Comment:
I decided to remove the urls from the .env files because this file is used by the script as a way to provide the user input data so we should not use it to pass the urls because it is our business to set them - .env file is for the user. We can always push a fresh commit updating hardcoded urls 😉 

However TBH I have a mixed feelings regarding the purpose of using .env file. IMO user input data should be passed as a script argument not as env variables. I think we should revisit that in the future.